cfgov/v1/jinja2/v1/includes/molecules/call-to-action.html

```diff
@@ -37,9 +39,12 @@ <h2 class="a-heading">
     </div>
     {% endif %}
 
-    {% if value.button %}
+    {% if value.button.url and value.button.text %}
     <a class="a-btn a-btn__full-on-xs"
-       href="{{ value.button.url or '/' }}">
+       href="{{ value.button.url or '/' }}"
```

Comment:
This looks like existing code, but isn't this href condition already guarded? `value.button.url` will never be falsy here

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/7968/commits/0c3044edd35c19c1eac09f2e02043f37e7df882e thanks!